lib/utils/getpackagenames.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const minimatch = require( 'minimatch' );
+
+/**
+ * @param {Options} options
```

Comment:
The options param in this function doesn't need to satisfy the `Options` interface.